bescort.lic

```diff
@@ -227,6 +227,7 @@ class Bescort
     ]
 
     args = parse_args(arg_definitions)
+    @flying_mount = get_settings.flying_mount
```

Comment:
New setting? Add it to base.yaml as well

Reply:
Good catch

Reply:
I'll add it.